src/mo_frontend/parser.mly

```diff
@@ -459,6 +459,7 @@ inst :
 %inline typ_params_opt :
   | (* empty *) { [] }
   | LT ts=seplist(typ_bind, COMMA) GT { ts }
+  | LT ASYNC GT { [{var = "" @@ at $sloc; sort = Type.Scope @@ no_region; bound = PrimT "Any" @! at $sloc} @= at $sloc] }
```

Comment:
is there a helper for this?

Reply:
also, allow more params?

Reply:
Needs a check that the function is `Local`.

Reply:
I think there is some helper for the scope parameter - look around... - maybe look at AsyncE

Reply:
Yes: b480313cd